mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -345,34 +364,48 @@ def _compute_moc_climo_postprocess(self):  # {{{
 
         config = self.config
 
-        dvEdge, areaCell, refBottomDepth, latCell, nVertLevels, \
-            refTopDepth, refLayerThickness = self._load_mesh()
-
         self.regionNames = config.getExpression(self.sectionName,
                                                 'regionNames')
 
         # Load basin region related variables and save them to dictionary
+        mpasMeshName = config.get('input', 'mpasMeshName')
```

Comment:
@milenaveneziani, please note that I made some significant changes starting here.  Please let me know if these changes make sense to you or if you have concerns.